parser/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -2,53 +2,58 @@ package hmda.parser.fi.lar
 
 import hmda.model.fi.lar._
 
-import scala.util.Try
+import scala.collection.immutable.ListMap
+import scalaz._
+import scalaz.Scalaz._
+import scala.util.{ Failure, Success, Try }
 
 object LarCsvParser {
   def apply(s: String): Either[List[String], LoanApplicationRegister] = {
```

Comment:
If this PR is accepted, then let's consider changing the return type here to use Validation stuff instead of `Either`. Two reasons:
* it'll make calling code, such as `LarHttpApi`, more consistent.
* no need to do a conversion here.

This question may be separate from the decision of whether to accept this PR into the larger one. If needed, I'm happy to re-make an appropriate comment there.